index.html

```diff
@@ -1655,9 +1655,9 @@ <h4 id="positioning_of_running_heads_and_page_numbers">
           <p its-locale-filter-list="ja" lang="ja"> 縦組の書籍又は横組の書籍において，ノンブル又は柱を横組にして配置する場合は，左ページでは，基本版面の左端の延長線にノンブル又は柱の先頭をそろえて配置するか，基本版面の左端の延長線から基本版面の文字サイズの全角アキだけ右に寄せた位置に配置する．右ページでは，基本版面の右端の延長線にノンブル又は柱の末尾をそろえて配置するか，基本版面の右端の延長線から基本版面の文字サイズの全角アキだけ左に寄せた位置にノンブル又は柱の末尾をそろえて配置する．</p>
         </li>
         <li id="id101">
-          <p its-locale-filter-list="en" lang="en">Regardless of the direction of text in a book, when arranging running heads and page numbers together on the same horizontal line, the space between the running head and the page number
+          <p its-locale-filter-list="en" lang="en">Regardless of the direction of text in a book, when arranging running heads and page numbers together on the same horizontal line (such as positioning running head at the very next to the page number at the left side on the left page), the space between the running head and the page number
```

Comment:
<p its-locale-filter-list="en" lang="en">Regardless of the writing direction of the text in a book, when arranging running heads and page numbers together on the same horizontal line (such as positioning the running head immediately adjacent to the page number at the left side on left pages), the space between the running head and the page number